src/frontend/komponenter/VedleggListe.tsx

```diff
@@ -45,12 +47,16 @@ const VedleggListe: React.FC<IOpplastedeVedlegg> = ({
     }
   };
 
+  const skalViseUndertittel = dokumenttype === 'Annet';
+
   return (
     <>
-      {vedleggsliste.map((fil: IVedleggForEttersending) => {
+      {vedleggsliste.map((fil: IVedleggForEttersending, index) => {
         return (
           <div key={fil.id}>
-            <Label as={'p'}>Dokumenter:</Label>
+            {skalViseUndertittel && index === 0 && (
```

Comment:
Skal den ikke vise "Dokumenter" for andre dokumenttyper enn 'Annet'? 👀

Reply:
skulle vært en `||`, fikser [e1165eb](https://github.com/navikt/familie-ef-ettersending/pull/523/commits/e1165ebbfe62864f2c44010c222b713658e590b1)